front/app/api/permissions/useUpdatePermission.ts

```diff
@@ -4,21 +4,25 @@ import { CLErrors } from 'typings';
 import fetcher from 'utils/cl-react-query/fetcher';
 
 import permissionKeys from './keys';
-import { IPhasePermission, IPermissionUpdate } from './types';
+import { IGlobalPermission, PermissionUpdateParams } from './types';
 
 const updatePermission = async ({
   action,
   ...requestBody
-}: Partial<IPermissionUpdate>) =>
-  fetcher<IPhasePermission>({
+}: Partial<PermissionUpdateParams>) =>
+  fetcher<IGlobalPermission>({
     path: `/permissions/${action}`,
     action: 'patch',
     body: { permission: requestBody },
   });
 
 const useUpdatePermission = () => {
```

Comment:
I get what you mean, but the path is `/permissions` and not `/global_permissions`... so then it would be inconsistent with that. Since we will get rid of this soon anyway I suggest we leave it like this